packages/fluentui/react-northstar/src/components/Input/Input.tsx

```diff
@@ -281,6 +295,8 @@ Input.propTypes = {
   type: PropTypes.string,
   value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
   wrapper: customPropTypes.wrapperShorthand,
+  required: PropTypes.bool,
```

Comment:
I added required here to be a handled prop, therefore not adding it to the DOM element as per spec. But like that I have no idea how we can make it clear it is a required field, my first thought was to used [`aria-required`](https://www.w3.org/TR/WCAG20-TECHS/ARIA2.html) but the spec also suggest against it. Didn't completely got the reason tho. @kolaps33 @jurokapsiar

Reply:
I probably did mistake in summary because based on Adam comment(see below) "aria-required" should behave differently and could be used. :)
Sorry about confusion. 

The “required” HTML attribute is not used on the form fields because on the form submission, when some required field is empty, the browser’s native alert message takes priority and makes the custom live region message not be spoken. The “aria-required” appears to behave differently, as it currently does not cause the browser’s native message be narrated upon a submission of a form with an empty required field.

Reply:
Great, added `aria-required` then.